windows/lt-base-windows-dotnet.json

```
@@ -51,17 +51,17 @@
       "type": "powershell",
       "inline": [
         "choco install 7zip --version 19.0",
-        "choco install nuget.commandline --version 5.3.0",
+        "choco install nuget.commandline --version 5.8.0",
         "choco install sonarscanner-msbuild-net46 --version=4.10.0.19059",
         "choco install sonarscanner-msbuild-netcoreapp2.0 --version=4.10.0.19059",
-        "choco install visualstudio2019buildtools --version=16.3.6.0",
+        "choco install visualstudio2019buildtools --version 16.8.2.0 --package-parameters \"--allWorkloads --includeRecommended --includeOptional --passive --locale en-US\"",
```

I think we can omit this parameter
```suggestion
        "choco install visualstudio2019buildtools --version 16.8.2.0 --package-parameters \"--allWorkloads --includeRecommended --includeOptional --passive\"",
```